src/parsers/js/acorn.js

```diff
@@ -24,16 +24,33 @@ export default {
   locationProps: new Set(['range', 'loc', 'start', 'end']),
 
   loadParser(callback) {
+    const parsers = {};
+    const _localReq = require;
+
     require(['acorn', 'acorn-jsx/inject'], (acorn, jsxInject) => {
```

Comment:
Also, we should probably do the same thing with acorn itself, but that doesn't have to be done in this PR.